v1/lists.yaml

```diff
@@ -251,7 +259,7 @@ paths:
       summary: Update a list.
       description: |
         List must belong to current user and request must be authenticated with
-        a MediaWiki session cookie.
+        a MediaWiki session cookie. If the name is changed, the new name must not be in use.
```

Comment:
Can we also specify what happens when there's a naming conflict? I guess 409 could be a nice response code

Reply:
It results in a `readinglists-db-error-duplicate-list` error. Is 409 really applicable? The conflict isn't really with the current status of the resource, it's with another resource. 409 seems more appropriate for things like an edit conflict.